mon-pix/app/components/choice-chip.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+
+export default class ChoiceChip extends Component {
+  get className() {
+    const classNames = ['choice-chip'];
+    this.args.isDisabled && classNames.push('choice-chip--disabled');
```

Comment:
Je préfère la clareté du `if`

```suggestion
    if (this.args.isDisabled) {
    	classNames.push('choice-chip--disabled');
    }
```